src/targets/gpu/lowering.cpp

```diff
@@ -43,6 +43,7 @@
 #include <utility>
 #include <functional>
 #include <algorithm>
+#include <rocblas.h>
```

Comment:
You should include `migraphx/gpu/rocblas.hpp` header instead.

Reply:
changed.